src/ui/ServerAdministration/index.tsx

```diff
@@ -86,6 +87,14 @@ class ServerAdministrationContainer
   /* A list of object schemas to use when creating the next Realm */
   protected createRealmSchema?: Realm.ObjectSchema[];
 
+  private readonly compatibilityVersions: Array<{
+    rosVersion: string;
+    studioVersion: string;
+  }> = [
+    { rosVersion: '2.0.0', studioVersion: '1.19.2-ros2' },
```

Comment:
Can we have a more recent Studio or do we really have to go back in time?

Reply:
Not sure I understand? ROS < 3.0 needs that ancient studio version. Or do you propose that we do a one-off studio release with a ROS 2.x compatible realm-js? (I also noticed this should be 3.0.0, not 2.0.0).

Reply:
If I understand correctly, that's what we're doing - ROS versions below 3.11.0 will prompt the user to downgrade to Studio 2.8.2 which was supposed to be the pinned equivalent of 2.8.0 before I realized there are unreleased changes in master 😓 Then if the ROS version is lower than 3.0.0, we'll prompt the user to downgrade to 1.19.2.

Reply:
:+1: